src/test/java/edu/kis/powp/jobs2d/events/SelectCommandVisitorCounterListener.java

```diff
@@ -0,0 +1,52 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.command.visitor.CommandVisitorCounter;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class SelectCommandVisitorCounterListener implements ActionListener {
+
+    private Logger logger = Logger.getLogger("global");
+    private DriverManager driverManager ;
+
+    public SelectCommandVisitorCounterListener(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        logger.info("Testing visitor for driver command.");
+
+        List<DriverCommand> commands = new ArrayList<DriverCommand>();
+        commands.add(new SetPositionCommand(-20, -50));
+        commands.add(new OperateToCommand(-20, -50));
+        commands.add(new SetPositionCommand(-20, -40));
+        commands.add(new OperateToCommand(-20, 50));
+        commands.add(new SetPositionCommand(0, -50));
+        commands.add(new OperateToCommand(0, -50));
+        commands.add(new SetPositionCommand(0, -40));
+        commands.add(new OperateToCommand(0, 50));
+        commands.add(new SetPositionCommand(70, -50));
+        commands.add(new OperateToCommand(20, -50));
+        commands.add(new OperateToCommand(20, 0));
+        commands.add(new OperateToCommand(70, 0));
+        commands.add(new OperateToCommand(70, 50));
+        commands.add(new OperateToCommand(20, 50));
```

Comment:
Please move this command to a separate test so you can set it using driverCommandManager.

In this test use getCurrentCommand to access a command to be processed by the visitor.